testsuite/features/build_validation/init_clients/proxy_container.feature

```diff
@@ -17,39 +18,40 @@ Feature: Setup Containerized Proxy
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
-  Scenario: Pre-requisite: Stop traditional proxy service
-    When I stop salt-minion on "proxy"
-    And I run "spacewalk-proxy stop" on "proxy"
-    # workaround for bsc#1205976
-    And I stop the "tftp" service on "proxy"
-    And I wait until "squid" service is inactive on "proxy"
-    And I wait until "apache2" service is inactive on "proxy"
-    And I wait until "tftp" service is inactive on "proxy"
+# TODO: Uncomment the following scenarios when this PR is merged: https://github.com/uyuni-project/uyuni/pull/8148
+#  Scenario: Bootstrap the proxy HostOS as a Salt minion
+#    When I follow the left menu "Systems > Bootstrapping"
+#    Then I should see a "Bootstrap Minions" text
+#    When I enter the hostname of "proxy" as "hostname"
+#    And I enter "22" as "port"
+#    And I enter "root" as "user"
+#    And I enter "linux" as "password"
+#    And I select "1-proxy_key" from "activationKeys"
+#    And I click on "Bootstrap"
+#    And I wait until I see "Bootstrap process initiated." text
+#
+#  Scenario: Wait until the proxy HostOS appears
+#    When I wait until onboarding is completed for "proxy"
 
   Scenario: Generate Containerized Proxy configuration
     When I generate the configuration "/tmp/proxy_container_config.tar.gz" of Containerized Proxy on the server
-    And I copy "/tmp/proxy_container_config.tar.gz" file from "server" to "proxy"
-    And I run "tar xzf /tmp/proxy_container_config.tar.gz -C /etc/uyuni/proxy/" on "proxy"
+    And I copy the configuration "/tmp/proxy_container_config.tar.gz" of Containerized Proxy from the server to the proxy
 
-  Scenario: Set-up the Containerized Proxy service to support Avahi
-    And I add avahi hosts in Containerized Proxy configuration
+  Scenario: Run a Containerized Proxy
+    When I run "mgrpxy install podman /tmp/proxy_container_config.tar.gz" on "proxy"
 
-  Scenario: Start Containerized Proxy services
-    When I start the "uyuni-proxy-pod" service on "proxy"
+  Scenario: Wait until Containerized Proxy service is active
     And I wait until "uyuni-proxy-pod" service is active on "proxy"
     And I wait until "uyuni-proxy-httpd" service is active on "proxy"
     And I wait until "uyuni-proxy-salt-broker" service is active on "proxy"
     And I wait until "uyuni-proxy-squid" service is active on "proxy"
     And I wait until "uyuni-proxy-ssh" service is active on "proxy"
     And I wait until "uyuni-proxy-tftpd" service is active on "proxy"
     And I wait until port "8022" is listening on "proxy"
-    And I wait until port "8080" is listening on "proxy"
+    # And I wait until port "8080" is listening on "proxy"
```

Comment:
Why is this commented out?

Reply:
Same question.

I suppose this port is not forwarded to the outside of the container. But then, what is it so? And what was its purpose?

Reply:
I need to research why it was not listening to that port.
I will add another TODO, but I don't want to block the PR just for that. I prefer to start getting  some feeling of the status of this integration.

Reply:
According to https://suma-bv-50-srv.mgr.suse.de/docs/en/suse-manager/installation-and-upgrade/proxy-container-installation.html#installation-proxy-containers-firewall-rules, port `8080` is not mentioned there.

Reply:
Checking on running beta-1 containers

host -> container

```
0.0.0.0:80->8080/tcp, 0.0.0.0:443->443/tcp, 0.0.0.0:4505-4506->4505-4506/tcp, 0.0.0.0:8022->22/tcp, 0.0.0.0:69->69/udp
```

Reply:
I will for now keep the other ports without a check, I will do another PR for that, after re-testing if they are available immediately.